colossalai/booster/plugin/hybrid_parallel_plugin.py

```diff
@@ -287,6 +314,13 @@ def __init__(self,
                                check_reduction=check_reduction,
                                gradient_as_bucket_view=gradient_as_bucket_view,
                                static_graph=static_graph)
+
+        self.zero_config = dict(reduce_bucket_size=bucket_cap_mb * 1024 * 1024,
```

Comment:
I remember `reduce_bucket_size` is the number of elements, rather than bytes. Please check this.

Reply:
Yes, `reduce_bucket_size` in Zero is in the number of elements. I have fixed this.